src/hyp3_isce2/utils.py

```diff
@@ -1,133 +1,17 @@
 import shutil
 import subprocess
-from dataclasses import dataclass
-from datetime import datetime
-from pathlib import Path
 from typing import Optional
 
 import isceobj
 import numpy as np
 from isceobj.Util.ImageUtil.ImageLib import loadImage
 from iscesys.Component.ProductManager import ProductManager
-from osgeo import gdal
+from osgeo import gdal, osr
 
 
 gdal.UseExceptions()
 
 
-class GDALConfigManager:
-    """Context manager for setting GDAL config options temporarily"""
-
-    def __init__(self, **options):
-        """
-        Args:
-            **options: GDAL Config `option=value` keyword arguments.
-        """
-        self.options = options.copy()
-        self._previous_options = {}
-
-    def __enter__(self):
-        for key in self.options:
-            self._previous_options[key] = gdal.GetConfigOption(key)
-
-        for key, value in self.options.items():
-            gdal.SetConfigOption(key, value)
-
-    def __exit__(self, exc_type, exc_val, exc_tb):
-        for key, value in self._previous_options.items():
-            gdal.SetConfigOption(key, value)
-
-
-@dataclass
-class ParameterFile:
```

Comment:
This gets called in `merge_tops_burst` along with `make_browse_image`. It might be worth leaving these functions in utils.py if we can.